private/tools/java/com/github/bazelbuild/rules_jvm_external/resolver/DependencyInfo.java

```diff
@@ -81,6 +82,18 @@ public SortedMap<String, SortedSet<String>> getServices() {
     return services;
   }
 
+  @Override
+  public String toString() {
```

Comment:
nit: use `MoreObjects#toStringHelper` which handles the formatting for you.

Reply:
Done.